packages/lyra/tests/lyra.test.ts

```diff
@@ -61,7 +61,7 @@ describe("checkInsertDocSchema", () => {
     ).toBeDefined();
 
     try {
-      await db.insert({ quote: "hello, world!", author: true });
+      await db.insert({ quote: "hello, world!", author: true } as any);
```

Comment:
Is there a way to avoid casting `as any`?

Reply:
I think any in these test cases is necessary because the document inserted does not match the schema used to create the DB. I can try to cast it to ‘PropertiesSchema’ but I am not sure if it works

Reply:
You're right, but then use // @ts-expect-error please 🙏

Thank you!